app/controllers/users_controller.rb

```diff
@@ -1,6 +1,9 @@
 class UsersController < ApplicationController
   def accept_transition_info
     User.member(current_user['user_id']).accept_transition_screen
```

Comment:
Does the client library allow adding logic to/around the `accept_transition_screen`? It'd be handy to somehow encapsulate the server 422 response within the `User` model, rather than let that bleed out into the controller. Just a thought...

Reply:
There's only the `on` and `request_method` options from what I gather: https://github.com/JsonApiClient/json_api_client/blob/e4b763f5579c4fae4b03dcc935b20627f5c14eaa/lib/json_api_client/resource.rb#L215